src/run.js

```diff
@@ -498,6 +501,10 @@ const minimalcss = async options => {
     // Here's the crucial part. Decide whether to keep the selector
     // Find at least 1 DOM that contains an object that matches
     // this selector string.
+    if (isInWhiteList(selectorString)) {
+      return true;
```

Comment:
Shouldn't this be added to `decisionCache`?
I.e. if you have `options.whitelist = ['.foo']` and something like:
```html
<p class="foo">One</p>
<p class="foo">Two</p>
```
it should only ever need to call `isInWhiteList` once. Right? 

It's been a while since I wrote the `decisionCache` implementation but the name is pretty clear :)
Can you take a look?

Reply:
Sure, in the current implementation, the result of the whole function isSelectorMatchToAnyElement is added to the decisionCache. And since we are doing whitelist checks inside this function, the result is being added to the decisionCache too.